draft-ietf-jsonpath-base.md

```diff
@@ -1911,9 +1921,6 @@ IANA is requested to register the following media type {{RFC6838}}:
 Type name:
 : application
 
-Subtype name:
-: jsonpath
-
 Required parameters:
```

Comment:
Not sure why this change is needed for this PR.

Reply:
"Subtype" is no longer used, so the definition isn't needed.  We don't have subtypes.  We have conversions.

"Subtype" implies an inheritance model.  We have three disparate types.

Reply:
This is a subtype of a media type in an IANA registration.
Fixed in #403.